.ci/run.sh

```diff
@@ -96,6 +96,11 @@ case "${CI_JOB}" in
 		echo "INFO: Running Kubernetes tests with Firecracker"
 		sudo -E PATH="$PATH" bash -c "make kubernetes"
 		;;
+	"FIRECRACKER_JAILER")
+		echo "INFO: Running Kubernetes tests with Firecracker (with jailer)"
+		sudo sed -i -e 's/#jailer_path/jailer_path/' "/usr/share/defaults/kata-containers/configuration.toml"
```

Comment:
Will it be possible to start another job afterwards ? Same CI environment, no reboot?
Is this change in the kata-configuration "safe" ?

Reply:
> Will it be possible to start another job afterwards ? Same CI environment, no reboot?

Another job from scratch, yes, I don't see a reason why we wouldn't.
Starting another job would, as the first thing, build & install kata-containers, then overriding this configuration file.

Does this make sense?

> Is this change in the kata-configuration "safe" ?

Define safe, as I'm not sure if I get what you mean.
By default, when using firecracker, this line **will** be present and **will** be commented out.  Not sure if this is what you mean by safe.

Reply:
I want to be able to re-use the CI env by simply changing env variables and then set.sh & run.sh.

Reply:
I mean setup.sh

Reply:
Well, you already answered, appreciated!

Reply:
So, `setup.sh`, if `INSTALL_KATA` is set to yes (which is the default) would install kata-containers, which would override the conifguration file change that we do here, see:
https://github.com/kata-containers/tests/blob/a1011147e583debe198c1b599dcb76803458cf9a/.ci/setup.sh#L93-L98

Is that enough for you?  Or would you prefer that just "undo" the sed after the test is done?  That's simple and if it makes you feel safer, why not?

Reply:
AFAIK setup.sh re-installs kata, so the config file that is being edited with this patch should be overwritten by the default. If kata is not re-installed with setup.sh then maybe we could re-comment the jailer option -- in any case, running *with* the jailer enabled could be the default option from now on ;-) I can't think of reason not to enable that...

Reply:
In any case, @ananos, @marcel-apf, give me your thumbs-up if you're fine with me squashing this patch to the original one:
```
fidencio@quino ~/go/src/github.com/kata-containers/tests $ git diff
diff --git a/.ci/run.sh b/.ci/run.sh
index 6374148a..75aaf08e 100755
--- a/.ci/run.sh
+++ b/.ci/run.sh
@@ -98,8 +98,10 @@ case "${CI_JOB}" in
                ;;
        "FIRECRACKER_JAILER")
                echo "INFO: Running Kubernetes tests with Firecracker (with jailer)"
-               sudo sed -i -e 's/#jailer_path/jailer_path/' "/usr/share/defaults/kata-containers/configuration.toml"
+               sudo sed -i -e 's/^#jailer_path/jailer_path/' "/usr/share/defaults/kata-containers/configuration.toml"
                sudo -E PATH="$PATH" bash -c "make kubernetes"
+               # revert to the current "default" behaviour, which is not using the jailer
+               sudo sed -i -e 's/^jailer_path/#jailer_path/' "/usr/share/defaults/kata-containers/configuration.toml"
                ;;
        "VFIO")
                echo "INFO: Running VFIO functional tests"
```

This should make the scenario of a re-run super safe, even in case someone sets `INSTALL_KATA` to `no`, while still working for our case.

Reply:
@fidencio appreciated, already acked, but anyway:
lgtm